src/PVRIptvData.cpp

```diff
@@ -469,12 +470,23 @@ bool PVRIptvData::LoadPlayList(void)
         tmpChannel.properties.insert({prop, propValue});
       }
     }
+    else if (StringUtils::Left(strLine, (int)strlen(PLAYLIST_TYPE_MARKER)) == PLAYLIST_TYPE_MARKER)
+    {
+      std::string value = ReadMarkerValue(strLine, PLAYLIST_TYPE_MARKER);
+      if (value == "EVENT")
+        tmpChannel.properties.insert({PVR_STREAM_PROPERTY_ISREALTIMESTREAM, "true"});
+      else if (value == "VOD")
+        tmpChannel.properties.insert({PVR_STREAM_PROPERTY_ISREALTIMESTREAM, "false"});
```

Comment:
No need to insert the prop if the value is false, as this is the default if the prop is not present.